chain/ethereum/src/sf/mod.rs

```diff
@@ -0,0 +1,277 @@
+pub mod pb {
```

Comment:
I am a big fan of short names, but this might be a little too short; maybe call it `protobuf`?